### DISCUSSION TITLE
MDF.configure(raise_on_multiple_occurrences=False) [v6.3.0]

### DISCUSSION THREAD

**Discussion by pebel76260** [danielhrisca/asammdf#542] [Q&A] [answered]
Hello @danielhrisca,

first of all, thank you for your great work!
Unfortunately, I have a lot of duplicate signals in my mf4-file. Version 6.2.0 raises exception on multiple occurrences. Despite I changed the way I filter and load the mf4 file (filter by channel group index tuple), I tried to set 
mdf_obj.configure(raise_on_multiple_occurrences=False), but this option seems to have no effect on the MDF.filter. The exception still occurs. 
Do I misuse this option? I'm using v6.3.0

**Comment by danielhrisca** [comment, marked as answer]
Hello @pebel76260 ,  there was a bug with setting the flag. Please try the latest development branch code (`6.3.0.dev50`)

**Comment by JulienGrv** [comment]
Hi @pebel76260, 

An MDF file may contain multiple channels with identical name but different values. Consider looking for the exact channel you want, I provided an example in [this discussion](https://github.com/danielhrisca/asammdf/discussions/493).
